Sources/Build/BuildPlan.swift

```diff
@@ -2056,7 +2041,7 @@ public class BuildPlan {
 
         // Plan products.
         for buildProduct in buildProducts {
-            try plan(buildProduct)
+            try plan(buildProduct as! ProductBuildDescription)
```

Comment:
safe?

Reply:
Yep, since we know this always to be successful.